python/Ganga/GPIDev/Lib/Registry/RegistrySlice.py

```diff
@@ -72,10 +80,8 @@ def do_collective_operation(self, keep_going, method, *args, **kwds):
 
     def ids(self, minid=None, maxid=None):
         "Get the list of job ids. 'minid' and 'maxid' specify optional (inclusive) slice range."
-        if maxid is None:
-            maxid = sys.maxsize
-        if minid is None:
-            minid = 0
+        maxid = maxid if maxid not None else sys.maxsize
+        minid = minid if minid not None else 0
```

Comment:
I'll admit that this PR looks more complicated than my simple example and I haven't touched the JobRegistry in ages but why do we use `sys.maxsize` here rather than just a `max` as in my example?